Podfile.lock

```diff
@@ -395,7 +395,7 @@ PODS:
   - WordPress-Aztec-iOS (1.19.3)
   - WordPress-Editor-iOS (1.19.3):
     - WordPress-Aztec-iOS (= 1.19.3)
-  - WordPressAuthenticator (1.24.1):
+  - WordPressAuthenticator (1.24.0-beta.22):
```

Comment:
I expect to see your version here (1.24.2). Also, I didn't expect to see other pod changes. Did you branch off of `release/15.7` or `develop`?

Reply:
Good catch.  I made a mistake when merging the latest from `release/15.7` and apparently picked the lockfile from develop instead.

It's now fixed.